tests/docker/storage.spec.js

```diff
@@ -0,0 +1,40 @@
+import {uploadFile, getFile} from '../../src/client/storage';
+import {delay} from '../../src/core/utils';
+import {get, post} from 'axios';
+
+beforeAll(() => {
+    jest.setTimeout(300000);
+});
+
+describe('Storage upload/download', () => {
+    let id;
+    it('Should upload a file', async () => {
+        id = await uploadFile('Test string');
+        expect(id).toBeTruthy();
+    });
+
+    it('Should download an uploaded file', async () => {
+        await delay(120000); // Waiting for the file to be mined
```

Comment:
Instead of just delay, let's poll `getFile` function once in 500ms or something until timeout. Then we can measure and see the actual time it takes to get mined.